### PR TITLE
Create sms-counter.user.js

### PR DESCRIPTION
**Pull request by plocket**
Allow user to see the number of sms messages the text they've entered will send.

Original is here: https://github.com/plocket/sms-counter

### CODE REVIEW

File: sms-counter.user.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    return;
+  }
+
+  // Otherwise add the message counter element to the page
+  var gyrContainer = document.querySelector( '.text-message-form .form-group' );
+  console.log(`The text area on the messages screen is: ${ gyrContainer }`);
+  if (gyrContainer === null) {
+    // Could be on the "Take Action" page
+    gyrContainer = document.querySelector( '#hub_take_action_form_message_body' ).parentElement;
+    console.log(`The text area on the Take Action screen is: ${ gyrContainer }`)
```
Comment (michaelaltmann) [inline sms-counter.user.js]: Maybe remove this debugging log

**Review by michaelaltmann — COMMENTED** [review]
There is something amiss with your math. At 159 characters it says 1 message and 1 char to go.  Then at 160 chars it says 1 message, 160 chars to go.  At 161 characters, it says 2 message and 133 to go.
I think the statement for 160 chars should say 1 message, 134 till the next.  
Screen Shot 2021-03-11 at 8 47 41 AM
Screen Shot 2021-03-11 at 8 47 52 AM
Screen Shot 2021-03-11 at 8 48 07 AM

**Comment by plocket** [comment]
> There is something amiss with your math.

1. Nice eye
2. That actually brings up a discussion topic. Other than some specific complications, 158 is the number that splits messages, but I think some services try to keep whole words as one piece. Should we be cautious and create a margin, or should we be exact?

**Comment by plocket** [comment]
I'm not sure the statement for 160 quite makes sense that way either now that I think about it. At 160 characters you have one full message. One more character will turn it into two messages. Other options:

#### Option 1
1 msgs (2 chars left)
Num chars 159

1 msgs (1 chars left)
Num chars 160

2 msgs (134 chars left)
Num chars 161

---

#### Option 2
1 msgs (1 chars left)
Num chars 159

1 msgs (0 chars left)
Num chars 160

2 msgs (133 chars left)
Num chars 161

---

Option 2 makes the most sense to me [at the moment].

**Comment by plocket** [comment]
I gave it a shot. Let me know what you think.

**Pull request merged**